Curve-finance-suite/Unkill/src/agent.spec.ts

```diff
@@ -0,0 +1,90 @@
+import {
+  FindingType,
+  FindingSeverity,
+  Finding,
+  HandleTransaction,
+} from "forta-agent";
+import { createAddress, TestTransactionEvent } from "forta-agent-tools";
+import { provideHandleTransaction, createFinding } from "./agent";
+import { stablePoolInterface } from "./abi";
+
+const POOL_OWNER = createAddress("0x1");
+
+describe("Unkill Agent Test Suite", () => {
+  let handleTransaction: HandleTransaction;
+
+  beforeAll(() => {
+    handleTransaction = provideHandleTransaction(POOL_OWNER);
+  });
+
+  it("should return empty findings if nothing happens", async () => {
```

Comment:
Maybe being more specific with what "nothing happens" means?

Change to something like `should return empty Findings because of non-relevant transactions`?